client/src/App/components/Main/RescueUpdates/Form/Form.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+
+import { addUpdate } from "../../../../../redux/actions/update";
+
+import isEmpty from "../../../../../utils/isEmpty";
+
+import {
+  FormContainer,
+  TextArea,
+  Row,
+  DateTimeInput,
+  Button
+} from "./Form-styles";
+
+const Form = () => {
+  const dispatch = useDispatch();
+  const { selectedRescue } = useSelector(state => state.rescue);
+
+  const [update, setUpdate] = useState("");
+  const [timestamp, setTimestamp] = useState("");
+  const [isValidTimestamp, setIsValidTimestamp] = useState(true);
+  const [isValidUpdate, setIsValidUpdate] = useState(true);
+
+  const setCurrentDateTime = () => {
+    const dt = new Date();
+
+    let day = dt.getDate();
+    let month = dt.getMonth() + 1;
+    const year = dt.getFullYear();
+
+    let hours = dt.getHours();
+    let minutes = dt.getMinutes();
+
+    if (month < 10) {
+      month = `0${month}`;
+    }
+
+    if (day < 10) {
+      day = `0${day}`;
+    }
+
+    if (hours < 10) {
+      hours = `0${hours}`;
+    }
+
+    if (minutes < 10) {
+      minutes = `0${minutes}`;
+    }
+
+    const today = `${year}-${month}-${day}T${hours}:${minutes}`;
```

Comment:
You can remove all of the above formatting by just doing dt.toISOString().slice(0, -8).
toISOString() function returns the format YYYY-MM-DDTHH:mm:ss.sssZ, so we slice it by 8 characters from the end to remove timezone, milliseconds and seconds.

Reply:
Will change implementation since input type datetime-local is not supported by other browsers